src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * Create migration for session table when pdo handler is used
  * Add support for Relay PHP extension for Redis
  * The `Response::sendHeaders()` method now takes an optional HTTP status code as parameter, allowing to send informational responses such as Early Hints responses (103 status code)
+ * Add `IpUtils::isPrivateIp`
```

Comment:
You need to rebase to solve a conflict in this file.

Reply:
Done. Not sure if correct though.